pvkrun.in.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec ENABLE_PRIMUS_LAYER=1 optirun "$@"
```

Comment:
Doesn't the environment variable need to be before `exec` for this to work?

Reply:
yes, fixed that.